canonical/ubuntu.com
# Replace custom macros with Vanilla patterns

## Done

- Replace custom macros in case studies with new Vanilla patterns
- Update macros usage for hero and quote sections

## QA

- Go to any case study, e.g https://ubuntu-com-14526.demos.haus/case-study/lucid-aws-fedramp-compliance-case-study
- Check that **hero** and **quote section** is similar to prod: https://ubuntu.com/case-study/lucid-aws-fedramp-compliance-case-study

## Issue / Card

Fixes [WD-17503](https://warthogs.atlassian.net/browse/WD-17503)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-17503]: https://warthogs.atlassian.net/browse/WD-17503?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
